.well-known/apple-app-site-association

```diff
@@ -54,11 +54,11 @@
                     },
                     {
                         "/": "/split/*",
-                        "comment": "Split Bill"
+                        "comment": "Split Expense"
                     },
                     {
                         "/": "/request/*",
-                        "comment": "Request Money"
+                        "comment": "Submit Expense"
```

Comment:
Why was this here twice? Now its going to have same `path` and `comment`, we can remove one right?